src/game.py

```diff
@@ -42,10 +45,19 @@ def place_mark_on_board(self, user_input, board, play_count):
     def process_user_input(self):
         position_choice = int(input())
         self.place_mark_on_board(position_choice, self.board, self.play_count)
+        self.play_count += 1
         print(self.initialize_board())
 
+    def progress_game(self):
+        current_play_count = self.count_plays()
+        while current_play_count != len(self.board):
```

Comment:
Whenever I write I while loop, I ask myself "is this something that I could do recursively instead?" Do you think this is something you could do recursively instead? Do you think this is something you should do recursively instead?